app/components/document-text.ts

```diff
@@ -125,9 +115,36 @@ class DocumentTextCtrl {
     return `${revision.remote.type}, ${revision.remote.id}`;
   }
 
-  updateActiveRevision(revisions) {
+  async isRevisionAccessible(revision) {
+    if (revision.isOpenAccess) {
+      return true;
+    }
+    if (revision.remote.type === 'elsevier') {
+      const result = await this.$http.get(
+        `https://api.elsevier.com/content/article/entitlement/doi/${revision.doi}`,
+        {
+          params: {
+            apiKey: 'd7cd85afb9582a3d0862eb536dac32b0',
```

Comment:
we have 2 options here:
 * move to `config.json`
 * extract `apiKey` from `revision.file.url`